pkg/multicluster/local/local_cluster_manager.go

```diff
@@ -0,0 +1,61 @@
+package local
+
+import (
+	"context"
+	"github.com/ibm/the-mesh-for-data/manager/apis/app/v1alpha1"
+	"github.com/ibm/the-mesh-for-data/pkg/multicluster"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	m4dSystemNs                  string = "m4d-system"
```

Comment:
change to a `namespace` parameter to the LocalClusterManager (because I think that we can eventually detect it, but definitely not part of the scope of local_cluster_manager)